cmake_modules/dependencies/yakkety.cmake.in

```diff
@@ -1,2 +1,3 @@
+#UBUNTU 16.04
```

Comment:
looks like we have #UBUNTU 16.04 here and in the zesty file. one should be 17.xx right?